build.xml

```diff
@@ -478,7 +478,7 @@ Please:
   <target name="validjavaversion"
           depends="buildtools" >
 
-    <property name="min.java.version" value="1.6.0_10" />
+    <property name="min.java.version" value="1.7.0_10" />
```

Comment:
I believe there was a specific reason (JVM bug) that the minimum was specifically 6u10, so changing the minimum from 6u10 to 7u10 is probably slightly wrong. But I'm not sure what the correct value should be.